_docs/concepts/traffic-management/rules-configuration.md

```diff
@@ -53,58 +62,91 @@ following important aspects must be kept in mind while writing route rules:
 
 Every rule corresponds to some destination service identified by a
 *destination* field in the rule. For example, all rules that apply to calls
-to the "reviews" service will include the following field.
+to the "reviews" service will include at least the following.
 
 ```yaml
-destination: reviews.default.svc.cluster.local
+destination:
+  name: reviews
 ```
 
-The *destination* value SHOULD be a fully qualified domain name (FQDN). It
-is used by Pilot for matching rules to services. For example,
-in Kubernetes, a fully qualified domain name for a service can be
-constructed using the following format: *serviceName.namespace.dnsSuffix*.
+The *destination* value specifies, implicitly or explicitly, a fully qualified
+domain name (FQDN). It is used by Pilot for matching rules to services.
+
+The FQDN of the service is composed from three components: *name*,
+*namespace*, and *domain*:
```

Comment:
You might want to specify the alternative as well. name+namespace+domain OR service (it works now. and thats what Consul and egress rules use).